### ISSUE TITLE
Transition "delete sprite" prompt to modal

### ISSUE THREAD

**Issue by thisandagain** [LLK/scratch-gui#1477]
### Expected Behavior

- Deleting a sprite should cause a prompt, but that prompt should be a React modal

### Actual Behavior

- We are using the "default" JS prompt

**Comment by hedgehog125** [comment]
Yes, also it shouldn't come up in fullscreen. What about on touchscreens though? Maybe a hold?
